x/incentives/keeper/distribute.go

```diff
@@ -426,7 +426,7 @@ func (k Keeper) Distribute(ctx sdk.Context, gauges []types.Gauge) (sdk.Coins, er
 			filteredLocks := k.getDistributeToBaseLocks(ctx, gauge, locksByDenomCache)
 			// send based on synthetic lockup coins if it's distributing to synthetic lockups
 			var err error
-			if lockuptypes.IsSyntheticDenom(gauge.DistributeTo.Denom) {
+			if lockuptypes.IsSyntheticDenom(gauge.DistributeTo.Denom) { // If the gauge is superfluid gauge,
```

Comment:
Was this comment intended to be inline?

Reply:
yes! I would often times prefer inline comments to if / else statements, but no strong preference here